addons/mail/static/src/js/client_action.js

```diff
@@ -201,23 +207,20 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
                                  'channel_inbox';
         var default_channel = chat_manager.get_channel(default_channel_id) ||
```

Comment:
rename default_channel into defaultChannel, and default_channel_id into defaultChannelID